README.md

```diff
@@ -107,6 +110,9 @@ You can also use the following default configurations:
     testnet=true
     paywallxpub=tpubVobLtToNtTq6TZNw4raWQok35PRPZou53vegZqNubtBTJMMFmuMpWybFCfweJ52N8uZJPZZdHE5SRnBBuuRPfC5jdNstfKjiAs8JtbYG9jx
     paywallamount=10000000
+    cachehost=politeiawww_user@localhost:26257/politeia
+    cacherootcert="/Users/<username>/Library/Application Support/cockroachdb/certs/ca.crt"
```

Comment:
I don't like the OSX specific dirs here at all. This should be UNIXy.

Reply:
Fixed.